.buildkite/common.py

```diff
@@ -32,11 +33,15 @@ def group(label, command, instances, platforms, agent_tags=None, **kwargs):
     # Use the 1st character of the group name (should be an emoji)
     label1 = label[0]
     steps = []
+    if isinstance(commands, str):
+        commands = [commands]
```

Comment:
I thought this was already in main? Maybe an artifact of a rebase...

Reply:
I had created the PR before this was mainlined. I'll do a cleanup of common.py